src/main.pl

```diff
@@ -0,0 +1,15 @@
+use 5.36.0;
+use strict;
+use warnings;
+use utf8;
+
+use Encode qw/decode_utf8/;
+use Data::Dumper;
+
+use lib '.';
+
+require 'lib/KamakuraTrash.pm';
+require 'lib/TrashInfoDeliver.pm';
```

Comment:
このケースだと、
```suggestion
use KamakuraTrash;
use TrashInfoDeliver;
```
と、`use`を使うほうが良さそうです。今は`require`はあまり使いません。`use`はBEGINフェーズで評価されるため、通常の文よりも先に動きます。またif文の中で条件分岐するのも効きません。
`use`と違い`require`は、プログラムの通常動作時に動くため、if文の中などでも意図通りに動作します。なので普段は`use`を使い、条件によってモジュールを追加で読み込みたいときなどに`require`を使いますが、基本的には`use`を使います。また、`import`関数が実行されるかされないかなどの違いもあります。